tests/test_propagate.py

```diff
@@ -1,24 +1,182 @@
+import numpy as np
+import pyarrow as pa
+import pyarrow.compute as pc
 import pytest
-from adam_core.orbits.query import query_sbdb
-from numpy.testing import assert_allclose
 from adam_core.coordinates.residuals import Residuals
+from adam_core.orbits.query.horizons import query_horizons
+from adam_core.orbits.query.sbdb import query_sbdb
+from adam_core.time import Timestamp
+from astropy import units as u
+from numpy.testing import assert_allclose
+
 from src.adam_core.propagator.adam_assist import (
     ASSISTPropagator,
     download_jpl_ephemeris_files,
 )
 
-def test_propagator_integration():
+DEFAULT_POSITION_TOLERANCE = (50 * u.m).to(u.au).value
+DEFAULT_VELOCITY_TOLERANCE = (1 * u.mm / u.s).to(u.au / u.day).value
+
+
+OBJECTS = {
+    "2020 AV2": {
+        "position": DEFAULT_POSITION_TOLERANCE,
+        "velocity": DEFAULT_VELOCITY_TOLERANCE,
```

Comment:
Not at all necessary but we could always parameterize using pytest and have each object be its own unit test. Would be pretty nifty to somehow include that testing infrastructure generically in adam_core and then we can use that to generate propagator comparisons. Way out of scope but just noting it here.

Reply:
I agree, I think having a library of objects for the test cases would be really beneficial. Not sure exactly what that looks like, and I need to button this up for now so that it can be used to debug impact study next week.